src/chains.ts

```diff
@@ -557,36 +558,67 @@ export const COSTON_TESTNET: Chain = {
         address: '0xc064943492c9DF4f8238Bf52E7B7170A0Ec6FBAF',
         active: true,
         reward_token: '0x0A8744c2a48b0e09a3e4C3A381A8E0d8d900bAFe',
-        type: StakingType.SAR_POSITIONS
-      }
-    ]
+        type: StakingType.SAR_POSITIONS,
+      },
+    ],
   },
   nativeCurrency: {
     name: 'CostonFlare',
     symbol: 'CFLR',
-    decimals: 18
+    decimals: 18,
   },
-  blockExplorerUrls: ['https://coston-explorer.flare.network']
+  blockExplorerUrls: ['https://coston-explorer.flare.network'],
 }
 
 export const EVMOS_TESTNET: Chain = {
   id: 'evmos_testnet',
   chain_id: 9000,
   name: 'EVMOS Testnet',
   symbol: 'tEVMOS',
+  png_symbol: 'evmPNG',
   mainnet: false,
   evm: true,
   logo: 'https://raw.githubusercontent.com/pangolindex/sdk/master/src/images/chains/evmos.png',
   pangolin_is_live: false,
   tracked_by_debank: false,
   supported_by_gelato: false,
   rpc_uri: 'https://eth.bd.evmos.dev:8545',
+  contracts: {
+    png: '0x171a6A80cdDDd8f69134ff0471523400f2C16ABe',
+    factory: '0x02fD35823a82b8f02e5F2d1a351807B9d1B38c3B',
+    router: '0x6b6D3049dD90E6Ecc84f8dca55f4A847b3a63fc2',
+    wrapped_native_token: '0xcF5ef8d007a616066e5eaEa0916592374a0F478D',
+    local_multisig: '0x90CE7198211D6B17be9fa5204701F9c9aCe2970b',
+    community_treasury: '0x3520e13c0E3f49Aa522dBD4477280fe3DF8B40fC',
+    treasury_vester: '0x6B77Cb69Dae236bC708d75B76356911D2D197601',
+    mini_chef: {
+      address: '0xA96b69EE04E33E1752b059a7a9B7C9FE2B3C93A9',
+      active: true,
+      type: ChefType.PANGO_CHEF,
+    },
+    airdrop: {
+      address: '0x2D1B541Cb20aD73217aC30cbA07C4571AF729Bc7',
+      active: false,
+      type: AirdropType.MERKLE_TO_STAKING,
+    },
+    timelock: '0xdA291D8daD1c55BBe828c91C58d16A523148bE11',
+    fee_collector: '0x0Da69A72C3875f74385BC1B633618e68AB2666F8',
+    multicall: '0x4fE8a78EB5bbDdC90942e13a09397Ee3CA7ed350',
+    staking: [
+      {
+        address: '0x997415e58dAEa9117027d55DAB7E765748C50834',
+        active: true,
+        reward_token: '0x171a6A80cdDDd8f69134ff0471523400f2C16ABe',
+        type: StakingType.SAR_POSITIONS,
+      },
+    ],
+  },
```

Comment:
out of above addresses I am not able to see below addresses in explorer: 
- local_multisig
- community_treasury
- treasury_vester
- mini_chef
- timelock
- staking

can you please confirm its deployed successfully ?

Reply:
Explorer is a bit laggy, I think. I've checked that all contract are deployed successfully. Send you an image on discord about it.